Code/Geometry/GridUtils.h

```diff
@@ -16,12 +16,23 @@ namespace RDGeom {
 class UniformGrid3D;
 class Point3D;
 
+//! calculate the tversky index between the shapes encoded on two grids
+/*!
+
+   tanimoto(S1,S2) =  | S1&S2 | / ( alpha * | S1 | + beta * | S2 | + | S1&S2 | )
```

Comment:
this should be, I think:
`tversky(S1,S2) =  | S1&S2 | / ( alpha * ( | S1 | - | S1&S2 | ) + beta * ( | S2 | - | S1&S2 | ) + | S1&S2 | )`